Library/ViewModels/RewardPledgeViewModel.swift

```diff
@@ -224,20 +230,29 @@ RewardPledgeViewModelOutputs {
       .map { AppEnvironment.current.currentUser }
       .skipRepeats(==)
 
-    let shippingRules = projectAndReward
-      .switchMap { (project, reward) -> SignalProducer<[ShippingRule], NoError> in
+    let shippingIsLoadingProperty = MutableProperty(false)
```

Comment:
typo in that, we wanna `filter { isTerminating }.mapConst(false)`, but im sure the tests woulda caught it!